vault/quotas/quotas_rate_limit_test.go

```diff
@@ -221,6 +222,9 @@ func TestRateLimitQuota_Update(t *testing.T) {
 	qm, err := NewManager(logging.NewVaultLogger(log.Trace), nil, metricsutil.BlackholeSink(), true)
 	require.NoError(t, err)
 
+	view := &logical.InmemStorage{}
+	require.NoError(t, qm.Setup(context.Background(), view, nil))
+
```

Comment:
Is this the only test that needs to be updated like this?

<img src="https://media2.giphy.com/media/xT0BKwUV9bLhr8btRK/giphy.gif"/>

Reply:
Yeah, looks like between this and the tests in `quotas_test.go` (below), we should be good!